docs/getting_started.md

```diff
@@ -81,7 +81,7 @@ Because the architecture in this example is relatively simple—you're just
 stacking layers—you can define the inlined submodules directly within the
 `__call__` method and wrap it with the
 [@compact](https://flax.readthedocs.io/en/latest/flax.linen.html#compact-methods)
-decorator.
+decorator. For information on the ```@compact``` decorator, see [```setup``` vs ```compact```](https://flax.readthedocs.io/en/latest/guides/setup_or_nncompact.html).
```

Comment:
Suggestion:

- In Markdown, maybe a single back tick would be enough 🤔  Though ``` seems to work.
- Add "guide".
- Maybe we can also remind users that these are part of the Flax Linen API, since "setup" etc are common terms/words.

```suggestion
decorator. To learn more about the Flax Linen `@compact` decorator, refer to the [`setup` vs `compact`](https://flax.readthedocs.io/en/latest/guides/setup_or_nncompact.html) guide.
```

LMKWYT!

Reply:
Thanks for the suggestion @8bitmp3! I added it in.